features/tags_and_wrangling/wrangling_guidelines.feature

```diff
@@ -8,10 +8,13 @@ Feature: Wrangling Guidelines
 
   Given I am logged in as a "tag_wrangling" admin
     And I am on the wrangling guidelines page
-  And I follow "New Wrangling Guideline"
+  Then I should see "Wrangling Guidelines"
```

Comment:
Could you make the line above this into "When I go to the wrangling guidelines page"? Having a "Then" right after a "Given" isn't the end of the world, but it is slightly nonstandard, and fixing it is an easy tweak.

I'd also mention it would be a good idea to add a selector when checking for the "Wrangling Guidelines" link to avoid any false test results due to the text appearing in multiple places on the page.